doc/user/spark.md

```diff
@@ -233,6 +253,18 @@ yarn.
 Then replicate `daos-site.xml`, `core-site.xml`, `yarn-site.xml` and
 `capacity-scheduler.xml` to other nodes.
 
+There is one pitfall that YARN runs with DAOS UNS URI, "daos:///uns-path",
+configured to `fs.defaultFS`. You may get an error complaining pool/container
+UUIDs cannot be found. It's because Hadoop considers the default filesystem is
+DAOS since you configured DAOS UNS URI. YARN has some working directories
+defaulting to local path without schema, like "/tmp/yarn", which is then
+constructed as "daos:///tmp/yarn". With this URI, Hadoop cannot connect to DAOS
+since no pool/container UUIDs can be found if daos-site.xml is not provided too
+.
+
+The simplest workaround for now is not set DAOS UNS URI to `fs.defaultFS`. Or
+you can find out all local working directories and prefix them with "file:///".
```

Comment:
How to find out all local working directories and prefix them with "file:///"? Is this possible?

Reply:
yes. Most of working dir is related to "hadoop.tmp.dir". And you also can find all defaults from https://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-common/yarn-default.xml.